lib/zendesk_apps_support/validations/translations.rb

```diff
@@ -13,13 +13,14 @@ class TranslationFormatError < StandardError
       end
 
       class << self
-        def call(package)
+        def call(package, opts = {})
           package.files.each_with_object([]) do |file, errors|
             path_match = TRANSLATIONS_PATH.match(file.relative_path)
             next unless path_match
             errors << locale_error(file, path_match[1]) << json_error(file) << format_error(file)
             next unless errors.compact.empty?
-            errors.push(*validate_marketplace_content(file, package)) if file.relative_path == 'translations/en.json'
+            errors.push(*validate_marketplace_content(file, package, opts[:skip_marketplace_translations])) \
```

Comment:
I think I'd prefer to see this as
```
if file.relative_path == 'translations/en.json'
  my_long_method_call_line
end
```
instead of having the broken line here.